server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -3467,27 +3467,8 @@ public void testHandleDocumentFailure() throws Exception {
                 assertNotNull(indexResult.getTranslogLocation());
                 engine.index(indexForDoc(doc2));
 
-                // test non document level failure is thrown
```

Comment:
Because the way it injected an error is no longer working and the test method says `testHandleDocumentFailure` so the non document level failure shouldn't be relevant to it.